components/tools/travis-build

```diff
@@ -123,18 +123,7 @@ build_cpp()
 integration()
 {
     fold start integration
-    if [ "x$TRAVIS_BRANCH" == "x" ]; then
-        FILES=$(git diff --name-only origin/develop...HEAD)
-    else
-        FILES=$(git diff --name-only HEAD...$TRAVIS_BRANCH)
-    fi
-    for x in $(echo "$FILES" | grep OmeroPy/test/integration); do
-        TEST=${x#components/tools/OmeroPy/}
-        export PYTHONPATH=$PYTHONPATH:../target/lib/python
-        ( cd components/tools/OmeroPy/;
-          python setup.py test -t $TEST -m "not broken"
-        )
-    done
```

Comment:
Is this covered in the `test-integration` target or something we want to come back to later ?

Reply:
`test-integration` is a superset of this garbage. Note also that `integration` was only ever used from omero-test-infra --> `travis-build`. The trade-off is that just building touched tests built _fast_ but not every PR touched the tests.